crates/bevy_render/src/lib.rs

```diff
@@ -85,6 +85,9 @@ use std::{
 #[derive(Default)]
 pub struct RenderPlugin {
     pub render_creation: RenderCreation,
+    /// If `true`, disables asynchronous pipeline compilation.
+    /// This has no effect on macOS, WASM, or without the `multi-threaded` feature.
+    pub sync_pipelines: bool,
```

Comment:
not sure what happened in the commits, but this one was missed

Reply:
bizarre. thanks.